cmd/commands/daemon/command.go

```diff
@@ -28,21 +28,14 @@ func NewCommand() *cli.Command {
 	var di cmd.Dependencies
 
 	return &cli.Command{
-		Name:      "run",
-		Usage:     "Runs Mysterium node",
+		Name:      "daemon",
+		Usage:     "Starts Mysterium Tequilapi service",
 		ArgsUsage: " ",
 		Action: func(ctx *cli.Context) error {
-			if err := di.Bootstrap(cmd.ParseFlagsNode(ctx)); err != nil {
-				return err
-			}
+			errorChannel := make(chan error)
+			StartDaemon(ctx, &di, errorChannel)
 
-			cmd.RegisterSignalCallback(utils.SoftKiller(di.Node.Kill))
-
-			if err := di.Node.Start(); err != nil {
-				return err
-			}
-
-			return di.Node.Wait()
+			return <-errorChannel
 		},
 		After: func(ctx *cli.Context) error {
 			err := di.Node.Kill()
```

Comment:
`di.Shutdown()` should be done somewhere. 
I suggest to pull `di.Bootstrap` to here also

Reply:
e.g. maybe we move/merge `StartDaemon()` helper to `di.Bootstrap()`

Reply:
Merged it to the `di.Bootstrap()`